airflow/macros/__init__.py

```diff
@@ -86,3 +87,32 @@ def _integrate_plugins():
                     "import from 'airflow.macros.[plugin_module]' "
                     "instead. Support for direct imports will be dropped "
                     "entirely in Airflow 2.0.".format(i=macro_name))
+
+
+def s3_location(schema, table, is_dest=True):
+    """
+    Return a s3 location based on schema and table name if it is in production or local file location.
+
+    @param str schema: Database name
+    @param str table: Table name
+    @param bool is_dest: the table is a destination table(true) or intermediate table(false).
```

Comment:
I think this is wrong. `is_dest` is not about destination or intermediate tables. It should be a different variable name that means something more along the lines of "is insert-overwrite" or "is create stage validate"

Reply:
change in https://github.com/lyft/etl/pull/10571